contracts/src/ethereum/TokenBridge.sol

```diff
@@ -52,7 +54,7 @@ contract TokenBridge is
     }
 
     function validateInitData(bytes calldata data) internal pure override {
-        require(data.length == 64, "ILLEGAL_DATA_SIZE");
+        require(data.length == 92, "ILLEGAL_DATA_SIZE");
```

Comment:
`validateInitData` isn't used?

Reply:
Its an override, its used by the Proxy contracts

Reply:
shouldn't the `data.length` be equal to `96` instead?

Reply:
It should be but the test fails with that, so I am trying to understand how its calculated